tools/objinfo.ml

```diff
@@ -32,19 +32,6 @@ let shape = ref false
 
 module Magic_number = Misc.Magic_number
 
-let input_stringlist ic len =
```

Comment:
input_stringlist was reversing the list and silently dropping anything after the last `\000`